src/test/kotlin/org/jitsi/jibri/service/impl/SipGatewayJibriServiceTest.kt

```diff
@@ -0,0 +1,164 @@
+package org.jitsi.jibri.service.impl
```

Comment:
Missing license header

Reply:
I had to add it in another PR, please review it https://github.com/jitsi/jibri/pull/396